### ISSUE TITLE
question about image caption file types

### ISSUE THREAD

**Issue by enkie358** [kijai/ComfyUI-FluxTrainer#72]
Hello,

I was looking at the args.json file for some comparison and troubleshooting and saw this line:
 
 _"caption_extension": ".caption",_

In my dataset, my images are tagged with a identically named _.txt_ . Does this line mean that I have inadvertently been training without image captions at all?

If so, how should I address this problem? Change my picture captions to _.caption_ or is there a setting that I need to change?

Thanks

**Comment by kijai** [comment]
This actually confused me too initially, there is a global `caption_extension` argument, and then dataset specific `caption_extension` argument. Since the beginning I've used the dataset specific argument and hard coded it as .txt

Later someone pointed this out as they realised their .caption -files were never used, and I've since added the option to set the desired extension to the very first dataset node, still defaulting to .txt.

So in your case there's no need to worry, .txt has always worked. I've also added a console log line that states how many captions were loaded.
